src/components/studies/app-design/WelcomeScreenMessagingSection.tsx

```diff
@@ -43,21 +42,31 @@ const useStyles = makeStyles(theme => ({
 type WelcomeScreenMessagingSectionProps = {
   appDesignProperties: StudyAppDesign
   setAppDesignProperties: Function
-  updateAppDesignInfo: Function
   saveLoader: boolean
   saveInfo: Function
   SimpleTextInputStyles: React.CSSProperties
+  onUpdate: Function
+  study: Study
```

Comment:
here and in the other controls. I don't think you should pass down the entire study object. You should only pass the things you are actually updating, welcomeScreenHeader, salutation etc.